cmd/utils/flags.go

```diff
@@ -102,6 +102,10 @@ var (
 		Usage: "Data directory for the databases and keystore",
 		Value: DirectoryString{common.DefaultDataDir()},
 	}
+	KeyStoreDirFlag = DirectoryFlag{
+		Name:  "keystore",
+		Usage: "Directory for the keystore",
```

Comment:
Could you refine this comment a bit so that it emphasizes that the default is within the datadir? Otherwise people may think that they need to explicitly set this. I.e.

`Usage: "Directory for the keystore (default = inside the datadir)",`